schemas/context/destination.schema.json

```diff
@@ -0,0 +1,79 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/destination",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Destinations",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:abstract": false,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/record"
+  ],
+  "description": "Destinations Schema.",
+  "definitions": {
+    "destination": {
+      "properties": {
+        "xdm:destinationID": {
+          "title": "Destination ID",
+          "type": "string",
+          "description": "Unique Identifier for destination flow"
+        },
+        "xdm:destinationName": {
+          "title": "Destination Name",
+          "type": "string",
+          "description": "Destination flow name"
+        },
+        "xdm:destinationPlatform": {
+          "title": "Destination Platform",
+          "type": "string",
+          "description": "The destination platform selected for activation flow."
+        },
+        "xdm:createTime": {
+          "title": "Create Time",
+          "type": "string",
+          "description": "The date and UTC time when the activation flow to the destination was createdByID"
+        },
+        "xdm:createdByID": {
+          "title": "Created User ID",
+          "type": "string",
+          "description": "ID of user who created the destination flow"
+        },
+        "xdm:destinationDescription": {
+          "title": "Destination Description",
+          "type": "string",
+          "description": "Destination flow description"
+        },
+        "xdm:destinationStatus": {
+          "title": "Destination Status",
+          "type": "string",
+          "description": "Status of destination flow",
+          "meta:enum": {
+                  "enabled": "Enabled and ready to use",
+                  "disabled": "Disabled and cannot be used"
+               }
+        },
+        "xdm:version": {
```

Comment:
What is the type here??   Since `xdm:version` is a standard type, you should refer to the standard definition.